sql/CMakeLists.txt

```diff
@@ -40,7 +40,8 @@ set(MOD_FILES
     updates/2.1.0--2.1.1.sql
     updates/2.1.1--2.2.0.sql
     updates/2.2.0--2.2.1.sql
-    updates/2.2.1--2.3.0.sql)
+    updates/2.2.1--2.3.0.sql
+    updates/2.3.0--2.3.1.sql)
```

Comment:
Should I expect something like this also for the downgrade path?

Reply:
Yes. The downgrade path need to be added after the release is tagged since the downgrade tests check out a tagged release, so it is not added here.

Reply:
Hmm.... let me check this again....

Reply:
Thanks for catching it. Had to update the downgrade list at this step as well.